exonum-java-binding/launcher-plugins/tests/test_data/proto_sources/service.proto

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package exonum.examples.timestamping;
+
+message Config {
+    string time_service_name = 2;
```

Comment:
Why the number is `2`?

Reply:
To match the example binary representation from Protobuf docs. I'll add an explanation here.

Reply:
The binary representation is unrelated to field numbers (but each field does include the field number, on top of the field _type_). The field number has nothing to do with the on-the wire encoding (the _type_ = string defines the wire encoding, the tag — identifies the field): https://developers.google.com/protocol-buffers/docs/proto3#assigning-field-numbers